content/docs/get-started/serving.md

```diff
@@ -9,7 +9,7 @@ serving models via REST API is quite common.
 To start up a FastAPI server run:
 
 ```cli
-$ mlem serve fastapi --model models/rf
+$ mlem serve models/rf fastapi
```

Comment:
I believe you're using MLEM < 0.3.0.

Reply:
```suggestion
$ mlem serve fastapi --model models/rf
```

Reply:
@aguschin you're right.
Not sure why 🤦  my bad